### ISSUE TITLE
Draft v0.92 2022-07-08

### ISSUE THREAD

**Issue by RogerLott** [opengeospatial/CRS-Gridded-Geodetic-data-eXchange-Format#35]
Updated draft. Includes decisions from discussions on NetCDF implementation, and changes to several keywords in the conventions. Examples have been updated for these changes.

**Comment by RogerLott** [comment]
[GGXF v0.92 spec 2022-07-08.pdf](https://github.com/opengeospatial/CRS-Gridded-Geodetic-data-eXchange-Format/files/9075373/GGXF.v0.92.spec.2022-07-08.pdf)

**Comment by desruisseaux** [comment]
Note: recommendations 15 and 16 at page 22 seem to duplicate recommendations 5 and 6 at page 15, or maybe I missed a nuance?

**Comment by desruisseaux** [comment]
In table C.2, for `affineCoeffs` definition, the following sentence:

> For a 3D grid the sequence of coefficients shall be A1, A2, A0, B1, B2, B0, C1, C2 and C0.

should be:

> For a 3D grid the sequence of coefficients shall be A1, A2, A3, A0, B1, B2, B3, B0, C1, C2, C3 and C0.

**Comment by desruisseaux** [comment]
* In table D.2, "Matrix element" column header, the prime symbol is missing for A1, A2, A0, B1, B2, B0.
* Typo in section E.1: "grid induces" → "grid indices" (I also saw "grup" instead of "group" in another section).

In the following sentence:

> because the rotation values in this affine transformation (the second and sixth coefficents) are zero

Instead of "rotation values", the term that I see used (e.g. in Java2D documentation from Oracle) is "shear values". I noticed that Wikipedia "World File" page used "rotation", but I think this is inaccurate because this value is not a rotation angle, or at least not directly. It is rather sin(θ).
